lib/repository.py

```diff
@@ -101,7 +101,7 @@ def run(self):
             scripts = []
             print("Downloading scripts...")
             for line in version_file:
-                scripts.append(line.strip('\n').split(','))
+                scripts.append(line.decode("utf-8").strip('\n').split(','))
```

Comment:
There is no error, just that `sys.setdefaultencoding` function is unavailable in python3.
https://docs.python.org/3/library/sys.html